docker/table-diff-handle/tablediffview/create_diff_markdown.py

```diff
@@ -0,0 +1,149 @@
+import csv
+import html2text
+import re
+from tablediffview.config import CONFIG
+from itertools import zip_longest
+from os import listdir
+from os.path import isfile
+from tabulate import tabulate
+
+
+class htmlcelldata:
+    def __init__(self, text: str, align: str) -> None:
+        self.text = text
+        self.align = align
+
+
+class diffmarkdown:
+    def __init__(self, test_file: str, table: str, md: str) -> None:
+        self.test_file = test_file
+        self.table = table
+        self.md = md
+
+
+def create_diffs(diff_dir: str) -> list[diffmarkdown]:
+    test_file_diffs = get_test_file_diffs(diff_dir)
+    csv_path_pattern = "{0}/{1}/{2}_{3}.csv"
+    mds: list[diffmarkdown] = []
+    for test_file, diff_tables in test_file_diffs.items():
+        for table_name in diff_tables:
+            print("Create diff view for: " + table_name)
+            with open(
+                csv_path_pattern.format(diff_dir, test_file, table_name, "current"),
+                encoding="utf-8",
+            ) as current:
+                with open(
+                    csv_path_pattern.format(
+                        diff_dir, test_file, table_name, "reference"
+                    ),
+                    encoding="utf-8",
+                ) as reference:
+                    diff_md = tabulate(
+                        create_diff_table(current, reference),
+                        headers="firstrow",
+                        tablefmt="github",
+                    )
+                    mds.append(diffmarkdown(test_file, table_name, diff_md))
+
+    return mds
+
+
+# Get the list of table, which haved changed
+def get_test_file_diffs(diff_dir: str) -> dict[str, set[str]]:
```

Comment:
@TruongQuangSB This function is a bit missleading because you are not getting the diffs. I would recommend to use `get_changed_test_files`

Reply:
ok